sprout/core/properties.py

```diff
@@ -413,7 +415,8 @@ class FieldProperties(Properties):
     missing_values: list[str] | list[MissingValueProperties] | None = None
 
     @classmethod
-    def default(cls: "type[FieldProperties]") -> "FieldProperties":
+    # use Type here to avoid error bc of variable `type`
+    def default(cls: Type[Self]) -> Self:
```

Comment:
```suggestion
    def default(cls: "type[Self]") -> Self:
```
This would be an alternative. Just because `Type` seems to be [deprecated](https://docs.python.org/3/library/typing.html#typing.Type)? But I'm not sure how much we care about that to be honest.